LibVLCSharp/Shared/Utf8StringMarshaler.cs

```diff
@@ -9,10 +9,11 @@ namespace LibVLCSharp.Shared
     /// <summary>
     /// Marshal unicode string param to utf-8 string,usage:[MarshalAs(UnmanagedType.CustomMarshaler, MarshalTypeRef = typeof(Utf8StringMarshaler))]
     /// </summary>
+#if !NETSTANDARD1_1
```

Comment:
Put this line above comment

Reply:
Why? Then the comment is not included and I have to have it twice

Reply:
You don't have a #else, so in case you're in netstandard, you have a comment on nothing

Reply:
> You don't have a #else

Yes, I do :-P It's just below, I think you missed it?

https://github.com/videolan/libvlcsharp/commit/2c63039b71c6e378e31d9a1b173dc599900a0d33#diff-60a9527f7fdf98b6c531858aea34e879

Will merge today unless you have more remarks